x-pack/plugins/cases/server/client/cases/create.test.ts

```diff
@@ -77,4 +77,36 @@ describe('create', () => {
       });
     });
   });
+
+  describe('Attributes', () => {
+    const clientArgs = createCasesClientMockArgs();
+    clientArgs.services.caseService.postNewCase.mockResolvedValue(caseSO);
+
+    beforeEach(() => {
+      jest.clearAllMocks();
+    });
+
+    it('should not have foo:bar attribute in request payload', async () => {
+      // @ts-expect-error
+      await create({ ...theCase, foo: 'bar' }, clientArgs);
+
+      expect(clientArgs.services.userActionService.creator.createUserAction).toHaveBeenCalledWith(
```

Comment:
Instead of checking for the not case how about we do `.toMatchInlineSnapshot` and just ensure that `foo` isn't in the result.